types/entria__relay-experimental/index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: https://github.com/facebook/relay
 // Definitions by: Renan Machado <https://github.com/renanmav>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.3
+// TypeScript Version: 3.7
```

Comment:
we can just remove entria__relay-experimental, we are not using anymore

Reply:
Did you mind doing that? I don't want to delay this pr any longer.

Reply:
So was this bump to deal with requirements of `entria__relay-experimental`, because in that case I’d prefer not to bump.

Reply:
The bump needs to happen because ts version before 3.7 doesn't support recursive types, and we need that for the EntryPoints

Reply:
Cool, got it 👍